tests/EvmAcceptanceTests/test/Subscriptions.js

```diff
@@ -0,0 +1,758 @@
+const {expect} = require("chai");
+const {ethers} = require("hardhat");
+
+const general_helper = require("../helper/GeneralHelper");
+
+// Ensures all events in current eventloop run are dispatched
+async function waitForEvents(events, timeout = 0) {
```

Comment:
let's not pollute the global scope. please add this function to `this` in `before` block. then you can use it like `this.waitForEvents` all over the tests